crates/storage/provider/src/providers/database/builder.rs

```diff
@@ -129,6 +155,16 @@ impl ReadOnlyConfig {
         Self::from_dirs(datadir.join("db"), datadir.join("static_files"))
     }
 
+    /// Disables long-lived read transaction safety guarantees, such as backtrace recording and
```

Comment:
Would remove backtrace recording notes because it's only relevant if the feature is enabled